<!DOCTYPE html>
<html lang="en">

<head>
    <title>Virology</title>
    <meta content="text/html; charset=utf-8" http-equiv="Content-Type">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="shortcut icon" sizes="152x152" href="/assets/images/logos/alexandrosMegas.jpg">
    <link href="/assets/main.css" media="screen" rel="stylesheet" type="text/css">
    <script src="/assets/main.js" type="text/javascript"></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=G-HBVJ1FF958"></script>
    <script>
        window.dataLayer = window.dataLayer || [];
        function gtag() { dataLayer.push(arguments); }
        gtag('js', new Date());

        gtag('config', 'G-HBVJ1FF958');
    </script>
</head>

<body class="">
    <div id="container">
        <div class="signature">
            <div class="flexcontainer">
                <div class="nav-view">
                    <div class="nav-inner">
                        <nav>
                            <a href="/" title="New Ideas"><img style="float: left;margin: 0px 35px 25px 35px;"
                                    src="/assets/images/logos/logo.svg" alt="website-logo" width="75" height="75"></a>

                            <ul>
                                <li><a href="/about" title="About">About</a></li>
                                <li><a href="/" title="Articles">Articles</a></li>
                                <li><a href="/research" title="Research">Research</a></li>
                                <li><a href="/reading" title="Reading">Reading</a></li>
                                <li><a href="/notes" title="Notes">Notes</a></li>
                            </ul>
                        </nav>
                        <div class="nav-extra-content">
                            <div class="divider"></div>
                            <p><em>Published:</em> March 11, 2024</p>
                            <p><em>Status:</em> Forgotten</p>
                        </div>
                    </div>
                </div>
                <div class="toc-index"></div>
                <div class="content">
                    <div class="signature-content-header">
                        <div id="title">
                            <h1>Virology</h1>
                            <h3>Lecture Notes of the 2020 Series on YouTube taught by Vincent Racaniello along with some
                                personal opinions.</h3>
                            <div class="publishing-info-mobile">
                                <p><em>Published:</em> March 11, 2024</p>
                                <p><em>Status:</em> Forgotten</p>
                            </div>
                        </div>
                    </div>
                    <div class="signature-content">
                        <div class="toc">
                            <h2>Contents</h2>
                            <ol>
                                <li><a href="#section-0">What is a Virus</a>
                                    <ol>
                                        <li><a href="#section-1">Course Goals</a></li>
                                        <li><a href="#section-2">What is a Virus</a></li>
                                        <li><a href="#section-3">Virus Classification</a></li>
                                        <li><a href="#section-4">Additional Reading</a></li>
                                    </ol>
                                </li>
                                <li><a href="#section-5">The Infectious Cycle</a>
                                    <ol>
                                        <li><a href="#section-6">How does one know if a particular cell is infected</a>
                                        </li>
                                        <li><a href="#section-7">How many viruses are there in the sample</a></li>
                                        <li><a href="#section-8">How many viruses are needed to form a plaque</a></li>
                                        <li><a href="#section-9">What do we do for viruses that do not form a plaque</a>
                                        </li>
                                        <li><a href="#section-10">Particle/PFU Ratio</a></li>
                                        <li><a href="#section-11">Single Step and Multi Step Growth Curves</a></li>
                                        <li><a href="#section-12">Additional Reading</a></li>
                                    </ol>
                                </li>
                                <li><a href="#section-13">Genomes and Genetics</a>
                                    <ol>
                                        <li><a href="#section-14">The Hershey-Chase Experiment</a></li>
                                        <li><a href="#section-15">Number of Viral Genomes</a></li>
                                        <li><a href="#section-16">What information is encoded in a viral genome</a></li>
                                        <li><a href="#section-17">What information is NOT encoded in a viral genome</a>
                                        </li>
                                        <li><a href="#section-18">Extra Reading</a></li>
                                    </ol>
                                </li>
                                <li><a href="#section-19">Structure of Viruses</a>
                                    <ol>
                                        <li><a href="#section-20">Functions of structural proteins</a></li>
                                        <li><a href="#section-21">Some definitions</a></li>
                                        <li><a href="#section-22">Metastability</a></li>
                                        <li><a href="#section-23">Symmetry</a></li>
                                        <li><a href="#section-24">Helical Symmetry</a></li>
                                        <li><a href="#section-25">Icosahedral Symmetry</a></li>
                                        <li><a href="#section-26">Enveloped Virions</a></li>
                                        <li><a href="#section-27">More Reading</a></li>
                                    </ol>
                                </li>
                                <li><a href="#section-28">Attachment and Entry</a>
                                    <ol>
                                        <li><a href="#section-29">How do viruses attach to receptors</a></li>
                                        <li><a href="#section-30">Some more reading</a></li>
                                    </ol>
                                </li>
                                <li><a href="#section-31">RNA directed RNA synthesis</a>
                                </li>
                            </ol>
                            <div class="divider-short"></div>
                        </div>
                        <article class="big">
                            <p><em>I got into learning some Virology during the COVID-19 pandemic and lost interest
                                    after a few lectures. Nevertheless, here is everything I learnt until now.
                                    <br><br>Note: The file was originally hosted on GitHub<input type="checkbox"
                                        id="cb-1" /><label for="cb-1"><sup id="footnote-1"><span
                                                class="node-link">1</span></sup></label><span><br><br>1: <a
                                            href="https://github.com/kyscg/Paper2Pulp/blob/master/notes/Virology with Vincent Racaniello.md">Virology
                                            with Vincent Racaniello</a></a><br><br></span></em>
                            </p>

                            <h2 id="section-0"><a href="#section-0">Lecture 1: What is a Virus</a></h2>
                            <ul>
                                <li>Just like we have a microbiome, we also have a virome.</li>
                                <li>The human genome has 3.2 billion bases out of which 8.3% are LTR transposons
                                    (remainders of retroviruses).</li>
                                <li>
                                    <p>Amazingly, the vast majority of the viruses that infect us have little to no
                                        impact on our health/well-being.
                                        Why not?</p>
                                    <ul>
                                        <li>
                                            <p>Most viruses just pass through us. (Many non-animal viruses are injested
                                                with food)</p>
                                        </li>
                                        <li>
                                            <p>Beneficial viruses.</p>
                                        </li>
                                        <li>We have an amazing immune system.</li>
                                    </ul>
                                </li>
                                <li>
                                    <p><a href="https://www.ncbi.nlm.nih.gov/pubmed/15831942"><strong>Use of
                                                polyomavirus to track <em>Homo
                                                    sapiens</em> migrations patterns in prehistoric times</strong></a>
                                    </p>
                                </li>
                            </ul>
                            <h3 id="section-1"><a href="#section-1">Course Goals</a></h3>
                            <ol>
                                <li>See the &#39;big picture&#39; of virology.</li>
                                <li>Virology is an integrated discipline, not an isolated collection of viruses,
                                    diseases and genes.</li>
                                <li>Fundamentals of the viruses.</li>
                            </ol>
                            <h3 id="section-2"><a href="#section-2">What is a Virus</a></h3>
                            <blockquote>
                                <p>A virus is an infectious, obligate, intracellular parasite comprising genetic
                                    material (DNA/RNA), often
                                    surrounded by a protein coat, sometimes a membrane.</p>
                            </blockquote>
                            <table>
                                <thead>
                                    <tr>
                                        <th>Term</th>
                                        <th>Meaning</th>
                                    </tr>
                                </thead>
                                <tbody>
                                    <tr>
                                        <td>Infectious</td>
                                        <td>Viruses can move from host to host.</td>
                                    </tr>
                                    <tr>
                                        <td>Obligate Intracellular</td>
                                        <td>Viruses have to get inside a cell in order to reproduce.</td>
                                    </tr>
                                    <tr>
                                        <td>Parasite</td>
                                        <td>Occupies a cell and takes essential stuff (nutrients), thereby damaging it.
                                        </td>
                                    </tr>
                                    <tr>
                                        <td>Protein shell/membrane</td>
                                        <td><strong>Often</strong> surrounded by a protein shell. Viroids don&#39;t have
                                            protein coats and some
                                            viruses (coronaviruses) have membranes</td>
                                    </tr>
                                </tbody>
                            </table>
                            <ul>
                                <li>
                                    <p>We need to avoid anthropomorphic analyses of viruses. They don&#39;t think, they
                                        don&#39;t have goals etc.
                                        <strong>The only selective force for a virus is to find a new host</strong>.
                                    </p>
                                </li>
                                <li>
                                    <p>Viruses are not all small. Some of the more recently discovered ones are
                                        relatively big.</p>
                                </li>
                                <li>
                                    <p>Viruses replicate by assembly of pre-formed components into many particles as
                                        opposed to binary fission like
                                        normal cells or bacteria. This gives rise to an incubation period where nothing
                                        seems to happen.</p>
                                </li>
                            </ul>
                            <table>
                                <thead>
                                    <tr>
                                        <th style="text-align:center"><img
                                                src="https://s3-us-west-2.amazonaws.com/courses-images/wp-content/uploads/sites/1094/2016/11/03154547/OSC_Microbio_06_02_growth.jpg"
                                                alt="Eclipse Period for Viruses"></th>
                                    </tr>
                                </thead>
                                <tbody>
                                    <tr>
                                        <td style="text-align:center"><em>Eclipse Period for Viruses</em></td>
                                    </tr>
                                </tbody>
                            </table>
                            <ul>
                                <li><strong>How old are viruses?</strong></li>
                            </ul>
                            <h3 id="section-3"><a href="#section-3">Virus Classification</a></h3>
                            <ul>
                                <li>
                                    <p>Nature and symmetry of nucleic acid in virion.</p>
                                </li>
                                <li>
                                    <p>Symmetry of protein shell (capsid).</p>
                                </li>
                                <li>
                                    <p>Presence/absence of lipid membrane (envelope).</p>
                                </li>
                                <li>
                                    <p>Dimensions of virus particle.</p>
                                </li>
                            </ul>
                            <p>There is an underlying simplicity and order to viruses because of two simple facts:</p>
                            <ol>
                                <li>
                                    <p>All viral genomes are obligate molecular parasites that can only function after
                                        they replicate in a cell.</p>
                                </li>
                                <li>
                                    <p>All viruses must make <code>mRNA</code> that can be translated by host ribosomes
                                        (they are all parasites of
                                        the host protein synthesis machinery).</p>
                                </li>
                            </ol>
                            <h3 id="section-4"><a href="#section-4">Additional Reading</a></h3>
                            <ol>
                                <li>Principles of Virology, Flint et al., Volume 1, Chapter 1</li>
                                <li><a href="https://www.virology.ws/2019/08/22/the-evolving-concept-of-virus/">The
                                        Evolving Concept of the
                                        Virus</a></li>
                                <li><a href="https://learn.genetics.utah.edu/content/cells/scale/">Cell Size and
                                        Scale</a></li>
                                <li><a
                                        href="https://www.virology.ws/2013/08/01/pandoravirus-bigger-and-unlike-anything-seen-before/">Pandoravirus</a>
                                </li>
                            </ol>
                            <h2 id="section-5"><a href="#section-5">Lecture 2: The Infectious Cycle</a></h2>
                            <p>The Infectious Cycle is everything that happens from when a virus attaches to a cell and
                                new viruses come out at the
                                end of the cycle. Other names: reproductive cycle, replicative cycle.</p>
                            <p>The steps of the infectious cycle are:</p>
                            <ol>
                                <li><strong>Attachment and entry</strong>: Viruses attach to very specific receptors on
                                    the cell surface and are
                                    brought into the cell.</li>
                                <li><strong>Translation</strong>: The <code>mRNA</code> is made and translated into
                                    proteins.</li>
                                <li><strong>Genome replication</strong>: Multiplication of viral genomes.</li>
                                <li><strong>Assembly</strong>: Viral proteins + viral genomes</li>
                                <li><strong>Release</strong>: Out in search of a new cell</li>
                            </ol>
                            <p>Some important definitions:</p>
                            <ul>
                                <li><strong>Susceptible cell</strong>: Cell has a functional receptor for a given virus
                                    - <em>the cell may or may
                                        not be able to support viral replication</em>.</li>
                                <li><strong>Resistant cell</strong>: Cell has no receptor - <em>it may or may not be
                                        able to support viral
                                        replication</em>.</li>
                                <li><strong>Permissive cell</strong>: Cell has the capacity to replicate virus - <em>it
                                        may or may not be
                                        susceptible</em>.</li>
                                <li><strong>Susceptible and Permissive cell</strong>: The only kinds of cells that can
                                    take a virus particle and
                                    replicate it.</li>
                            </ul>
                            <table>
                                <thead>
                                    <tr>
                                        <th style="text-align:center"><img
                                                src="https://s3-us-west-2.amazonaws.com/courses-images/wp-content/uploads/sites/1094/2016/11/03154532/OSC_Microbio_06_02_RepAnVirus.jpg"
                                                alt="Virus Infectious Cycle"></th>
                                    </tr>
                                </thead>
                                <tbody>
                                    <tr>
                                        <td style="text-align:center"><em>Virus Infectious Cycle</em></td>
                                    </tr>
                                </tbody>
                            </table>
                            <h3 id="section-6"><a href="#section-6">How does one know if a particular cell is
                                    infected</a></h3>
                            <p>One way is to look at them through a light microscope and looking for the <a
                                    href="https://en.wikipedia.org/wiki/Cytopathic_effect"><em>cytopathic
                                        effect</em></a> (CPE).</p>
                            <table>
                                <thead>
                                    <tr>
                                        <th style="text-align:center"><img
                                                src="https://www.researchgate.net/profile/Amr_Gamil2/publication/275258175/figure/fig1/AS:294613517651971@1447252744541/Cytopathic-effects-in-CHSE-cells-Development-of-cytopathic-effect-in-CHSE-214-cells.png"
                                                alt="HeLa Cells infected with Poliovirus"></th>
                                    </tr>
                                </thead>
                                <tbody>
                                    <tr>
                                        <td style="text-align:center"><em>HeLa cells infected with polioviruses, note
                                                the structural changes with
                                                time</em></td>
                                    </tr>
                                </tbody>
                            </table>
                            <p>Another way is to look for the formation of the <a
                                    href="https://en.wikipedia.org/wiki/Syncytium"><em>syncytia</em></a>. These are just
                                two ways of identification
                                and there are many others depending on the type of virus particle involved.</p>
                            <h3 id="section-7"><a href="#section-7">How many viruses are there in the sample</a></h3>
                            <p>We&#39;ve now identified that the virus is replicating and isolated the supernatant. How
                                do we know how much virus is
                                there in the supernatant?</p>
                            <ul>
                                <li>
                                    <p>There are two ways to do this, we can either measure infectivity or we can
                                        measure physical properties of
                                        virus particles and their components.</p>
                                </li>
                                <li>
                                    <p>We can use a <a
                                            href="https://en.wikipedia.org/wiki/Virus_quantification#Plaque_assay"><em>plaque
                                                assay</em></a> and quantify the virus. <em>Note: The Wikipedia link has
                                            all that we need to know
                                            about the method and reagants used.</em> The agar is important because it
                                        restricts the movement of the
                                        virus, if we just had fluid, the virus would move too fast for us to view the
                                        plaque formation (restricts
                                        viral diffusion after lysis of infected cells).</p>
                                </li>
                            </ul>
                            <p>Sometimes, the viruses don&#39;t kill the cells and we can&#39;t see the plaques. So we
                                add a gene to their genome
                                that adds a colour and count the coloured plaques instead.</p>
                            <h3 id="section-8"><a href="#section-8">How many viruses are needed to form a plaque</a>
                            </h3>
                            <p>If we are going to measure the virus titer by a plaque assay, we have to make sure that
                                one virus will make a plaque.
                                We use a dose-response curve to solve this problem.</p>
                            <table>
                                <thead>
                                    <tr>
                                        <th style="text-align:center"><img
                                                src="https://www.virology.ws/wp-content/uploads/2009/07/dose-response-plaque-assay-1018x1024.jpg"
                                                alt="Dose-Response Curve"></th>
                                    </tr>
                                </thead>
                                <tbody>
                                    <tr>
                                        <td style="text-align:center"><em>Dose-Response Curve</em></td>
                                    </tr>
                                </tbody>
                            </table>
                            <h3 id="section-9"><a href="#section-9">What do we do for viruses that do not form a
                                    plaque</a>
                            </h3>
                            <ul>
                                <li>
                                    <p>There is one method called the Endpoint dilution assay. Serial dilutions of a
                                        virus stock are prepared and
                                        inoculated onto replicate cell cultures, the number of cell cultures that are
                                        infected is then determined
                                        for each virus dilution, usually by looking for cytopathic effect.</p>
                                </li>
                                <li>
                                    <p>After an incubation period, plates that display cytopathic effects are scored
                                        with a <code>+</code>. At high
                                        dilutions, none of the cell cultures are infected because no particles are
                                        present. At low dilutions, every
                                        cell culture is infected. We need to find the dilution of virus at which 50% of
                                        the cell cultures are
                                        infected.</p>
                                </li>
                            </ul>
                            <p>The results are expressed as 50% lethal dose (LD<sub>50</sub>) per ml or 50% paralytic
                                dose (PD<sub>50</sub>) per ml
                                when lethality or paralysis are used as end points.</p>

                            <h3 id="section-10"><a href="#section-10">Particle/PFU Ratio</a>
                                <ul>
                                    <li>Particle/PFU Ratio = # of physical particles / # of infectious particles</li>
                                </ul>
                                <p>The term particle/PFU ratio refers to the number of viral particles required to form
                                    one
                                    plaque in a plaque assay. It
                                    is a measure of the efficiency by which a virus infects cultured cells. <strong>Not
                                        all
                                        virus particles are
                                        infectious</strong>. For example, the poliovirus has a ratio between 30 to 1000
                                    while Semliki forest virus has a
                                    ratio of 1 or 2.</p>
                                <ul>
                                    <li>
                                        <p>A single particle <em>can</em> initiate infection.</p>
                                    </li>
                                    <li>
                                        <p>Not all viruses are successful though</p>
                                        <ul>
                                            <li>Damaged particles (broken during assembly et al.)</li>
                                            <li>Mutations</li>
                                        </ul>
                                    </li>
                                </ul>
                                <strong>INCUBATION PERIOD AND ECLIPSE PERIOD ARE NOT THE SAME </strong>
                                <h3 id="section-11"><a href="#section-11">Single Step and Multi Step Growth Curves</a>
                                </h3>
                                <p>If we infect all the cells in one go, the virus will have a traditional growth
                                    curve
                                    with
                                    an eclipse period and
                                    stuff. But what happens in real life is that the virus particles never attack
                                    all
                                    the
                                    cells at one time. So we have,
                                    in reality, a multi-step growth curve that has multiple bursts with multiple
                                    eclipse
                                    periods.</p>
                                <ul>
                                    <li>Synchronous infection is the key to one-step growth cycles. But how do we
                                        know
                                        if
                                        we&#39;ve infected all the
                                        cells?</li>
                                </ul>
                                <h4 id="multiplicity-of-infection-moi-">Multiplicity of Infection (MOI)</h4>
                                <ul>
                                    <li>Number of infectious particles added per cell.</li>
                                    <li>Amount of virus (PFU) / # of cells</li>
                                </ul>
                                <p>This is different from the number of particles each cell recieves. If
                                    10<sup>7</sup>
                                    virus particles are added to
                                    10<sup>6</sup> cells. We have an MOI of 10, but each cell doesn&#39;t get 10
                                    virus
                                    particles.</p>
                                <ul>
                                    <li>Infection depends on the random collision of virus particles and cells. Best
                                        described by the Poisson
                                        Distribution.</li>
                                </ul>
                                <h3 id="section-12"><a href="#section-12">Additional Reading</a></h3>
                                <ol>
                                    <li>Principles of Virology, Flint et al., Volume 1, Chapter 2</li>
                                    <li><a href="https://www.virology.ws/2009/12/10/influenza-virus-growth-in-eggs/">Influenza
                                            virus growth in eggs</a>
                                    </li>
                                    <li><a
                                            href="https://www.virology.ws/2009/02/09/the-amazing-hela-cells-of-henrietta-lacks/">The
                                            amazing HeLa cells
                                            of Henrietta Lacks</a></li>
                                    <li><a href="https://www.youtube.com/watch?v=tGVDYGxqtJ0">Counting
                                            Viruses</a>
                                    </li>
                                    <li><a href="https://www.virology.ws/2017/02/17/viral-rna-is-not-infectious-virus/">Viral
                                            RNA is not infectious
                                            virus!</a></li>
                                </ol>
                                <h2 id="section-13"><a href="#section-13">Lecture 3: Genomes and Genetics</a></h2>
                                <p><a href="https://www.biorxiv.org/content/10.1101/2020.01.22.914952v1">Discovery
                                        of a
                                        novel coronavirus associated
                                        with the recent pneumonia outbreak in humans and its potential bat
                                        origin</a>.
                                    <em>The following line is from
                                        the paper, where we can see real-time applications of what we learned in
                                        the
                                        previous lecture</em>
                                </p>
                                <blockquote>
                                    <p>We then successfully isolated the virus (named nCoV-2019 124
                                        BetaCoV/Wuhan/WIV04/2019), in Vero and Huh7 cells
                                        using BALF sample from ICU-06 patient. Clear cytopathogenic effects were
                                        observed in
                                        cells after three days 126
                                        incubation.</p>
                                </blockquote>
                                <h3 id="section-14"><a href="#section-14">The Hershey-Chase Experiment</a></h3>
                                <p>Initially, the scientists studying virus particles thought that proteins
                                    carried
                                    genetic
                                    information. This was based
                                    on the belief that proteins were more complex than <code>DNA</code>. <em>For
                                        example,
                                        <code>DNA</code> in humans
                                        have only 4 bases, how could it carry that much of information</em>.</p>
                                <ul>
                                    <li>
                                        <p>Viruses were known to be composed of a protein shell and
                                            <code>DNA</code>, so
                                            they chose to uniquely label
                                            each with a different elemental isotope.
                                        </p>
                                    </li>
                                    <li>
                                        <p>Since phosphorus is contained in <code>DNA</code> but not amino
                                            acids,
                                            radioactive phosphorus-32 was used to
                                            label the <code>DNA</code> contained in the T2 phage. Radioactive
                                            sulfur-35
                                            was
                                            used to label the protein
                                            sections of the T2 phage, because sulfur is contained in protein but
                                            not
                                            <code>DNA</code>.
                                        </p>
                                    </li>
                                </ul>
                                <table>
                                    <thead>
                                        <tr>
                                            <th style="text-align:center"><img
                                                    src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/56/Tevenphage.svg/330px-Tevenphage.svg.png"
                                                    alt="Structural Overview of T2 Phage"></th>
                                        </tr>
                                    </thead>
                                    <tbody>
                                        <tr>
                                            <td style="text-align:center"><em>Structural Overview of T2
                                                    Phage</em>
                                            </td>
                                        </tr>
                                    </tbody>
                                </table>
                                <ul>
                                    <li>
                                        <p>Hershey and Chase inserted the radioactive elements in the
                                            bacteriophages
                                            by
                                            adding the isotopes to separate
                                            media within which bacteria were allowed to grow for 4 hours before
                                            bacteriophage introduction.</p>
                                    </li>
                                    <li>
                                        <p>When the bacteriophages infected the bacteria, the progeny contained
                                            the
                                            radioactive isotopes in their
                                            structures.</p>
                                    </li>
                                    <li>
                                        <p>This procedure was performed once for the sulfur-labeled phages and
                                            once
                                            for
                                            phosphorus-labeled phages.</p>
                                    </li>
                                    <li>
                                        <p>The labeled progeny were then allowed to infect unlabeled bacteria.
                                            The
                                            phage
                                            coats remained on the outside
                                            of the bacteria, while genetic material entered.</p>
                                    </li>
                                    <li>
                                        <p>Disruption of phage from the bacteria by agitation in a blender
                                            followed
                                            by
                                            centrifugation allowed for the
                                            separation of the phage coats from the bacteria. These bacteria were
                                            lysed
                                            to
                                            release phage progeny.</p>
                                    </li>
                                </ul>
                                <p><em>Note: We now know that some phages completely enter the bacteria like
                                        <code>E. Coli</code> but it was a fortunate
                                        coincidence that the T2 phages didn&#39;t do so.</em></p>
                                <ul>
                                    <li><strong>The progeny of the phages that were labeled with radioactive
                                            phosphorus
                                            remained labeled, whereas the
                                            progeny of the phages labeled with radioactive sulfur were
                                            unlabeled</strong>.
                                        Thus, the Hershey–Chase
                                        experiment helped to confirm that <code>DNA</code>, not protein, is the
                                        genetic
                                        material.</li>
                                </ul>
                                <table>
                                    <thead>
                                        <tr>
                                            <th style="text-align:center"><img
                                                    src="https://upload.wikimedia.org/wikipedia/commons/thumb/3/32/Hershey_Chase_experiment.png/536px-Hershey_Chase_experiment.png"
                                                    alt="Hershey-Chase Experiment"></th>
                                        </tr>
                                    </thead>
                                    <tbody>
                                        <tr>
                                            <td style="text-align:center"><em>Hershey-Chase Experiment</em></td>
                                        </tr>
                                    </tbody>
                                </table>
                                <p>Similar Experiment: The Fraenkel-Conrat Experiment on Tobacco Mosaic Virus.
                                </p>
                                <h3 id="section-15"><a href="#section-15">Number of Viral Genomes</a></h3>
                                <p>There are thousands of different virions, and it seems that there is an
                                    infinite
                                    complexity of infections. But the
                                    number of viral genomes is finite, in fact, it is my favourite number:
                                    <strong>7</strong>.
                                </p>
                                <p>One of the reasons for this is all viruses or viral genomes have to make
                                    <code>mRNA</code> that can be translated by
                                    the host ribosomes. This constraint causes the limit. <em>I have to admit,
                                        this
                                        doesn&#39;t exactly give a proper
                                        answer. What is the relation between the constraint on the
                                        <code>mRNA</code>
                                        and
                                        the
                                        number of viral genomes? I
                                        couldn&#39;t find an answer anywhere and I&#39;ve written a mail to
                                        Prof.
                                        Racaniello. I&#39;ll update it once I
                                        get an answer.</em>
                                </p>
                                <h4
                                    id="-the-baltimore-system-simplifying-virus-classification-https-www-virology-ws-2009-08-12-simplifying-virus-classification-the-baltimore-system-">
                                    <a
                                        href="https://www.virology.ws/2009/08/12/simplifying-virus-classification-the-baltimore-system/">The
                                        Baltimore
                                        System: Simplifying Virus Classification</a>
                                </h4>
                                <p>Virologist <a href="https://en.wikipedia.org/wiki/David_Baltimore">David
                                        Baltimore</a>
                                    devised an alternative
                                    classification scheme that takes into account the nature of the viral
                                    nucleic
                                    acid.
                                </p>
                                <p>All viruses <strong>must</strong> direct the synthesis of <code>mRNA</code>
                                    to
                                    produce
                                    proteins. No viral genome
                                    encodes a complete system for translating proteins; therefore all viral
                                    protein
                                    synthesis is completely dependent
                                    upon the translational machinery of the cell.</p>
                                <table>
                                    <thead>
                                        <tr>
                                            <th style="text-align:center"><img
                                                    src="https://www.virology.ws/wp-content/uploads/2009/08/baltimore-classification.jpg"
                                                    alt="Baltimore System"></th>
                                        </tr>
                                    </thead>
                                    <tbody>
                                        <tr>
                                            <td style="text-align:center"><em>Baltimore System</em></td>
                                        </tr>
                                    </tbody>
                                </table>
                                <table>
                                    <thead>
                                        <tr>
                                            <th>Class</th>
                                            <th>Type</th>
                                            <th>Remarks</th>
                                        </tr>
                                    </thead>
                                    <tbody>
                                        <tr>
                                            <td>1</td>
                                            <td>Double-stranded <code>DNA</code></td>
                                            <td>We can make <code>mRNA</code> from this</td>
                                        </tr>
                                        <tr>
                                            <td>2</td>
                                            <td>Single-stranded <code>DNA</code></td>
                                            <td>Double-stranded <code>DNA</code> is made from this and then to
                                                <code>mRNA</code>.
                                                <strong>Single-stranded <code>DNA</code> is not a template for
                                                    <code>mRNA</code> synthesis</strong>
                                            </td>
                                        </tr>
                                        <tr>
                                            <td>3</td>
                                            <td>Double-stranded <code>RNA</code></td>
                                            <td>Both the messenger and anti-messenger <code>RNA</code></td>
                                        </tr>
                                        <tr>
                                            <td>4</td>
                                            <td><code>+ RNA</code></td>
                                            <td>In some cases, we can translate it directly but we need to go
                                                through a
                                                <code>- RNA</code> to make more
                                                of it.
                                            </td>
                                        </tr>
                                        <tr>
                                            <td>5</td>
                                            <td><code>- RNA</code></td>
                                            <td>It must first be transcribed into a positive-sense
                                                <code>RNA</code>
                                                that
                                                acts as an <code>mRNA</code>.
                                            </td>
                                        </tr>
                                        <tr>
                                            <td>6</td>
                                            <td><code>+ RNA</code></td>
                                            <td>Unusual in the sense that it is converted to <code>DNA</code>
                                                which
                                                is
                                                then
                                                converted to double-stranded
                                                <code>DNA</code>
                                            </td>
                                        </tr>
                                    </tbody>
                                </table>
                                <p><em>Note: When originally conceived, the Baltimore scheme encompassed six
                                        classes
                                        of
                                        viral genome, as shown in the
                                        figure. Subsequently the gapped <code>DNA</code> genome of
                                        hepadnaviruses
                                        (e.g.
                                        hepatitis B virus) was
                                        discovered.</em></p>
                                <p>A few definitions that will help make more sense follow:</p>
                                <ul>
                                    <li><code>mRNA</code> is always the <code>+</code> strand.</li>
                                    <li><code>DNA</code> of equivalent polarity is also the <code>+</code>
                                        strand.
                                    </li>
                                    <li><code>RNA</code> and <code>DNA</code> complements of <code>+</code>
                                        strands
                                        are
                                        <code>-</code> strands.
                                    </li>
                                    <li>Not all <code>+ RNA</code> is <code>mRNA</code>!</li>
                                </ul>
                                <h4 id="in-conclusion-the-seven-classes-of-viral-genomes">In conclusion: The
                                    Seven
                                    Classes
                                    of Viral Genomes</h4>
                                <ul>
                                    <li>
                                        <p>Double Stranded <code>DNA</code></p>
                                    </li>
                                    <li>
                                        <p>Gapped Double Stranded <code>DNA</code></p>
                                    </li>
                                    <li>
                                        <p>Single Stranded <code>DNA</code></p>
                                    </li>
                                    <li>
                                        <p>Double Stranded <code>RNA</code></p>
                                    </li>
                                    <li>
                                        <p>Single Stranded <code>+ RNA</code></p>
                                    </li>
                                    <li>
                                        <p>Single Stranded <code>- RNA</code></p>
                                    </li>
                                    <li>
                                        <p>Single Stranded <code>+ RNA</code> with <code>DNA</code> intermediate
                                        </p>
                                    </li>
                                </ul>
                                <table>
                                    <thead>
                                        <tr>
                                            <th style="text-align:center"><img
                                                    src="https://microbeonline.com/wp-content/uploads/2016/06/Baltimore-System-of-Classification-of-Viruses.bmp"
                                                    alt="Baltimore System"></th>
                                        </tr>
                                    </thead>
                                    <tbody>
                                        <tr>
                                            <td style="text-align:center"><em>Baltimore System with
                                                    Examples</em>
                                            </td>
                                        </tr>
                                    </tbody>
                                </table>
                                <h3 id="section-16"><a href="#section-16">What information is encoded in a viral
                                        genome</a></h3>
                                <p>Gene products and regulatory signals for:</p>
                                <ul>
                                    <li>Protein synthesis (mainly giant viruses)</li>
                                    <li>Replication of the viral genome</li>
                                    <li>Assembly and packaging of the genome</li>
                                    <li>Regulation and timing of the replication cycle</li>
                                    <li>Modulation of host defenses</li>
                                    <li>Spread to other cells and hosts.</li>
                                </ul>
                                <h3 id="section-17"><a href="#section-17">What information is NOT encoded in a viral
                                        genome</a></h3>
                                <ul>
                                    <li>No genes encoding the complete protein synthesis machinery</li>
                                    <li>No genes encoding proteins involved in membrane biosynthesis</li>
                                    <li>No classical centromeres or telomeres found in standard host chromosomes
                                    </li>
                                    <li>Probably we haven’t found them yet - 90% of giant virus genes are novel.
                                    </li>
                                </ul>
                                <h3 id="section-18"><a href="#section-18">Extra Reading</a></h3>
                                <ol>
                                    <li>Principles of Virology, Flint et al., Volume 1, Chapter 3</li>
                                    <li><a
                                            href="https://www.virology.ws/2009/08/12/simplifying-virus-classification-the-baltimore-system/">Baltimore
                                            Classification</a></li>
                                    <li><a href="https://viralzone.expasy.org/">Viral Zone</a></li>
                                </ol>
                                <h2 id="section-19"><a href="#section-19">Lecture 4: Structure of Viruses</a></h2>
                                <h3 id="section-20"><a href="#section-20">Functions of structural proteins</a></h3>
                                <h4 id="protection-of-the-genome">Protection of the genome</h4>
                                <ul>
                                    <li>Assembly of a stable protective protein shell.</li>
                                    <li>Specific recognition and packaging of the nucleic acid genome.</li>
                                    <li>Interaction with host cell membranes to form the envelope.</li>
                                </ul>
                                <h4 id="delivery-of-the-genome">Delivery of the genome</h4>
                                <ul>
                                    <li>Bind host cell receptors.</li>
                                    <li>Uncoating of the genome.</li>
                                    <li>Fusion with cell membranes.</li>
                                    <li>Transport of genome to the appropriate site.</li>
                                </ul>
                                <h3 id="section-21"><a href="#section-21">Some definitions</a></h3>
                                <ul>
                                    <li><strong>Subunit</strong>: A single folded polypeptide chain. <em>Denoted
                                            by
                                            VP1,
                                            VP2
                                            etc.</em></li>
                                    <li><strong>Structural unit</strong> (protomer, asymmetric unit) - Unit from
                                        which
                                        capsids or nucleocapsids are
                                        built; one or more subunits.</li>
                                    <li><strong>Capsid</strong> (capsa = Latin, box) - Protein shell surrounding
                                        genome.
                                        <em>It is made up of
                                            subunits.</em>
                                    </li>
                                    <li><strong>Envelope</strong> (viral membrane) - Host cell-derived lipid
                                        bilayer.
                                        <em>For some viruses only, this
                                            always comes from the host cell because viruses cannot make
                                            lipids</em>.
                                    </li>
                                    <li><strong>Nucleocapsid</strong> (core) - Nucleic acid - protein assembly
                                        within
                                        particle. <em>Nucleocapsid is used
                                            against capsid when the it&#39;s a discrete substructure</em>.</li>
                                </ul>
                                <h3 id="section-22"><a href="#section-22">Metastability</a></h3>
                                <p>An important concept to understand is that virus particles are meta-stable.
                                    The
                                    capsid
                                    proteins have to be very
                                    stable in transit as the virus tries to find a new host <em>(Example:
                                        Respiratory
                                        virus
                                        particles have to be very
                                        stable in aerosols for effective transfer)</em> but when it finds the
                                    right
                                    host,
                                    the capsid has to give up the
                                    genome by becoming unstable.</p>
                                <p>The metastability of the virions (virus particles) arises from the fact that
                                    they
                                    haven&#39;t attained the minimum
                                    free energy conformation. They first need to surmount the energy barrier and
                                    the
                                    potential energy is used for
                                    disassembly if the host cell gives a proper signal.</p>
                                <table>
                                    <thead>
                                        <tr>
                                            <th style="text-align:center"><img
                                                    src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a0/Meta-stability.svg/375px-Meta-stability.svg.png"
                                                    alt="Metastability"></th>
                                        </tr>
                                    </thead>
                                    <tbody>
                                        <tr>
                                            <td style="text-align:center"><em>Metastability of virions (Giving
                                                    up of
                                                    genomes
                                                    happens at stage 3)</em>
                                            </td>
                                        </tr>
                                    </tbody>
                                </table>
                                <h4 id="how-is-metastability-achieved">How is metastability achieved</h4>
                                <ul>
                                    <li>
                                        <p>Stability in structure is achieved by creating a
                                            <strong>symmetrical</strong>
                                            arrangement of many identical
                                            proteins to
                                            provide maximal contact.
                                        </p>
                                    </li>
                                    <li>
                                        <p>Unstability happens primarily because these proteins aren&#39;t
                                            covalently
                                            joined
                                            and can be taken
                                            apart/loosened upon infection.</p>
                                    </li>
                                </ul>
                                <h3 id="section-23"><a href="#section-23">Symmetry</a></h3>
                                <p>There are primarily three kinds of virus particles: particles with helical
                                    symmetry
                                    (<em>Tobacco Mosaic Virus</em>),
                                    icosahedral symmetry (<em>Poliovirus</em>), complex (<em>don&#39;t fit into
                                        the
                                        above
                                        two</em>).</p>
                                <p>Let&#39;s go back to Watson and Crick who are most famous for discovering the
                                    structure
                                    of <code>DNA</code>. <em>For
                                        those of you who are into this stuff, I recommend reading The Double
                                        Helix,
                                        it&#39;s
                                        just fantastic</em>.</p>
                                <p>Watson and Crick observed that most virus particles were spherical or
                                    rod-shaped.
                                    They
                                    also knew that viral genomes
                                    were small which meant that particles would be built with many copies of a
                                    few
                                    proteins.
                                </p>
                                <ul>
                                    <li>
                                        <p>Rod-shaped viruses are identical protein subunits that are
                                            distributed
                                            with
                                            <em>helical symmetry</em>.
                                        </p>
                                    </li>
                                    <li>
                                        <p>Spherical viruses are identical protein subunits that are distributed
                                            with
                                            <em>icosahedral symmetry</em>.
                                        </p>
                                    </li>
                                </ul>
                                <h4 id="rules-of-symmetry">Rules of Symmetry</h4>
                                <ul>
                                    <li>
                                        <p><strong>Rule 1</strong>: Each subunit has ‘identical’ bonding
                                            contacts
                                            with
                                            its
                                            neighbors. (<em>Repeated
                                                interaction of chemically complementary surfaces at the subunit
                                                interfaces
                                                naturally leads to a
                                                symmetric arrangement</em>)</p>
                                    </li>
                                    <li>
                                        <p><strong>Rule 2</strong>: These bonding contacts are usually
                                            non-covalent.
                                            (<em>Reversible; error-free
                                                assembly. This means that if there&#39;s an error in the
                                                assembly of
                                                the
                                                particle, we can reverse it and
                                                re-assemble which wouldn&#39;t be possible in case of a covalent
                                                bond.</em>)
                                        </p>
                                    </li>
                                </ul>
                                <h4 id="symmetry-and-self-assembly">Symmetry and Self-Assembly</h4>
                                <p>If we have capsid proteins lying around in a cell, they will self-assemble
                                    into
                                    virus-like particles (VLP&#39;s) and
                                    these VLP&#39;s are used in some cases as vaccines. Some examples are the
                                    Hepatitis
                                    B
                                    vaccines, human papilloma
                                    virus vaccines which are VLP&#39;s made in yeast.</p>
                                <h3 id="section-24"><a href="#section-24">Helical Symmetry</a></h3>
                                <ul>
                                    <li>
                                        <p>The coat protein subunits form a helical subunit with number of
                                            proteins/turn
                                            =
                                            3. There are both
                                            protein-protein contacts as well as protein-<code>RNA</code>
                                            contacts.
                                            (<code>+ RNA</code> in case of TMV)
                                        </p>
                                    </li>
                                    <li>
                                        <p>Animal viruses also have helical symmetry but are always enveloped.
                                        </p>
                                    </li>
                                </ul>
                                <table>
                                    <thead>
                                        <tr>
                                            <th style="text-align:center"><img
                                                    src="http://www.twiv.tv/wp-content/uploads/2009/07/helical-symmetry2-1024x775.jpg"
                                                    alt="Helical Symmetry in TMV"></th>
                                        </tr>
                                    </thead>
                                    <tbody>
                                        <tr>
                                            <td style="text-align:center"><em>Helical Symmetry in TMV and the
                                                    envelope
                                                    in
                                                    animal viruses</em></td>
                                        </tr>
                                    </tbody>
                                </table>
                                <h3 id="section-25"><a href="#section-25">Icosahedral Symmetry</a></h3>
                                <p>The important question here is: How can you make a round capsid from proteins
                                    with
                                    irregular shapes? Watson and Crick
                                    answered that by using two important ideas:</p>
                                <ul>
                                    <li>
                                        <p>All round capsids have precise numbers of proteins; multiples of 60
                                            are
                                            common
                                            (60, 180, 240, 960)</p>
                                    </li>
                                    <li>
                                        <p>Spherical viruses come in many sizes, but capsid proteins are 20-60
                                            kDa
                                            average.
                                        </p>
                                    </li>
                                </ul>
                                <p>An icosahedron is a solid with 20 faces, each an equilateral triangle. It has
                                    five,
                                    three
                                    and two-fold axes of
                                    symmetry (12 each). It also allows formation of a closed shell with smallest
                                    number
                                    (60)
                                    of identical subunits.</p>
                                <h4 id="simple-icosahedral-capsids">Simple icosahedral capsids</h4>
                                <table>
                                    <thead>
                                        <tr>
                                            <th style="text-align:center"><img
                                                    src="http://www.twiv.tv/wp-content/uploads/2009/07/icosahedral-symmetry-1024x775.jpg"
                                                    alt="Icosahedral symmetry"></th>
                                        </tr>
                                    </thead>
                                    <tbody>
                                        <tr>
                                            <td style="text-align:center"><em>Icosahedral Symmetry</em></td>
                                        </tr>
                                    </tbody>
                                </table>
                                <ul>
                                    <li>Made of 60 identical protein subunits</li>
                                    <li>The protein subunit is the structural unit</li>
                                    <li>Interactions of all molecules with their neighbors are identical
                                        (head-to-head,
                                        tail-to-tail)</li>
                                    <li>The particles are spherical, not icosahedra! (<em>This means that the
                                            particles
                                            are
                                            just arranged in an
                                            icosahedral symmetry but they aren&#39;t flat and triangular, so
                                            they
                                            look
                                            more
                                            spherical</em>)</li>
                                </ul>
                                <h4 id="-t-triangulation-number-http-www-virology-wisc-edu-virusworld-tri_number-php-">
                                    <a href="http://www.virology.wisc.edu/virusworld/tri_number.php">T
                                        (Triangulation)
                                        Number</a>
                                </h4>
                                <p>It is the number of facets in each of the faces of this particle.</p>
                                <h3 id="section-26"><a href="#section-26">Enveloped Virions</a></h3>
                                <p>We know that capsids can be covered by host membranes. The envelope is a
                                    lipid
                                    bilayer
                                    derived from host cell as the
                                    viral genome does not encode lipid synthetic machinery.</p>
                                <p>Envelopes are usually acquired by budding of nucleocapsid through a cellular
                                    membrane.
                                    Can be any cell membrane, but
                                    is virus-specific.</p>
                                <p>Nucleocapsids inside the envelope may have helical or icosahedral symmetry.
                                </p>
                                <h3 id="section-27"><a href="#section-27">More Reading</a></h3>
                                <ol>
                                    <li>Principles of Virology, Flint et al., Volume 1, Chapter 4</li>
                                    <li><a href="https://youtu.be/qLAEUvlVmqY">Buckyball viruses</a></li>
                                    <li><a href="https://www.virology.ws/2009/03/06/virus-images-at-viperdb/">Virus
                                            images
                                            at VIPERdb</a></li>
                                </ol>
                                <h2 id="section-28"><a href="#section-28">Lecture 5: Attachment and Entry</a></h2>
                                <ul>
                                    <li>
                                        <p>Viruses are obligate intracellular parasites but they&#39;re too big
                                            to
                                            just
                                            diffuse across the plasma
                                            membrane.</p>
                                    </li>
                                    <li>
                                        <p>They need to find the <em>right</em> cell receptor. They first adhere
                                            to
                                            the
                                            cell
                                            surface via random
                                            collisions and electrostatics, then they attach to specific receptor
                                            molecules
                                            on cell surface and transfer
                                            the genome inside the cell.</p>
                                    </li>
                                </ul>
                                <p>There are some criteria for identifying cell receptors for viruses:</p>
                                <ul>
                                    <li>
                                        <p>We have to show that the receptor actually binds to the virus
                                            particle.
                                        </p>
                                    </li>
                                    <li>
                                        <p>There exists an antibody to the receptor that blocks infection.</p>
                                    </li>
                                    <li>
                                        <p>There is a receptor gene that confers susceptibility and its
                                            disruption
                                            blocks
                                            infection.</p>
                                    </li>
                                </ul>
                                <p>Sometimes, different viruses may bind to the same receptor. And other times,
                                    viruses
                                    of
                                    the same family bind to
                                    different receptors.</p>
                                <h3 id="section-29"><a href="#section-29">How do viruses attach to receptors</a></h3>
                                <p>For foreign particles, entry into viruses happens in many ways like <a
                                        href="https://en.wikipedia.org/wiki/Phagocytosis">phagocytosis</a>, <a
                                        href="https://en.wikipedia.org/wiki/Endocytosis">endocytosis</a> and <a
                                        href="https://www.frontiersin.org/articles/10.3389/fmicb.2016.00300/full#h6">surfing</a>.
                                    We are mostly
                                    interested in endocytosis.</p>
                                <p>Important point: <strong>To reiterate, the cytoplasm is extremely crowded,
                                        and
                                        movement
                                        of large protein complexes
                                        will not take place via diffusion</strong></p>
                                <h4 id="class-1-fusion-proteins">Class 1 Fusion Proteins</h4>
                                <ul>
                                    <li>
                                        <p>Perpendicular to membrane - spikes.</p>
                                    </li>
                                    <li>
                                        <p>Mostly α-helical.</p>
                                    </li>
                                    <li>
                                        <p>Form trimers.</p>
                                    </li>
                                </ul>
                                <h4 id="class-2-fusion-proteins">Class 2 Fusion Proteins</h4>
                                <ul>
                                    <li>
                                        <p>Mostly β-sheet.</p>
                                    </li>
                                    <li>
                                        <p>Form dimers.</p>
                                    </li>
                                    <li>
                                        <p>Parallel to the membrane.</p>
                                    </li>
                                </ul>
                                <h4 id="regulation-of-fusion">Regulation of fusion</h4>
                                <ul>
                                    <li>
                                        <p>Fusion must not occur in the wrong location.</p>
                                    </li>
                                    <li>
                                        <p>Neutral pH (plasma membrane)</p>
                                        <ul>
                                            <li>Second protein receptor interaction.</li>
                                        </ul>
                                    </li>
                                    <li>
                                        <p>Low pH fusion.</p>
                                        <ul>
                                            <li>
                                                <p>Proteolytic cleavage activates the fusion protein for
                                                    cleavage.
                                                </p>
                                            </li>
                                            <li>
                                                <p>Endosome fusion receptor.</p>
                                            </li>
                                        </ul>
                                    </li>
                                </ul>
                                <h3 id="section-30"><a href="#section-30">Some more reading</a></h3>
                                <ol>
                                    <li>Principles of Virology, Flint et al., Volume 1, Chapter 5</li>
                                    <li><a href="https://www.virology.ws/2019/02/14/a-portal-for-rna-exit/">A
                                            portal
                                            for
                                            RNA
                                            exit</a></li>
                                    <li><a href="https://www.virology.ws/2018/02/01/a-human-rhinovirus-in-chimpanzees/">A
                                            human rhinovirus in
                                            chimpanzees</a></li>
                                </ol>
                                <h3 id="section-31"><a href="#section-31">Lecture 6: RNA directed RNA synthesis</a></h3>
                                <div id="footnotes">
                                    <p>1: <a
                                            href="https://github.com/kyscg/Paper2Pulp/blob/master/notes/Virology%20with%20Vincent%20Racaniello.md">Virology
                                            Lectures by Vincent Racaniello</a>
                                        <script src="https://utteranc.es/client.js" repo="kyscg/kyscg.github.io"
                                            issue-term="url" label="comments" theme="github-light"
                                            crossorigin="anonymous" async>
                                            </script>

                                </div>
                                <div class="divider-short"></div>
                                <div class="info">
                                    <div class="publishing-info-mobile">
                                        <p><em>Published:</em> March 11, 2024</p>
                                        <p><em>Status:</em> Forgotten</p>
                                    </div>
                                </div>
                        </article>
                    </div>
                </div>
            </div>
        </div>
    </div>
</body>

</html>