<html>

<head>
    <title>Virology [UFO]</title>
    <meta charset='UTF-8'>
    <meta content='width=device-width, initial-scale=1' name='viewport' />

    <meta name='description' content=''>
    <meta name='keywords' content='notes, biology'>
    <meta name='author' content='Kilaru Yasaswi Sri Chandra Gandhi'>

    <link rel='shortcut icon' href='/assets/images/alexandrosMegas.jpg'>
    <link href='/index.css' rel='stylesheet'>

    <script>
  window.MathJax = {
    tex: {
      inlineMath: [['$', '$'], ['\\(', '\\)']]
    },
    svg: {
      fontCache: 'global'
    }
  };
</script>
<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-svg.js"></script>
</head>

<body>
    <div id="dappled-light">

    <div id="glow"></div>
    <div id="glow-bounce"></div>
    <div class="perspective">
        <div id="leaves">
            <svg style="width: 0; height: 0; position: absolute;">
                <defs>
                    <filter id="wind" x="-20%" y="-20%" width="140%" height="140%">
                        <feTurbulence type="fractalNoise" numOctaves="2" seed="1">
                            <animate attributeName="baseFrequency" dur="16s" keyTimes="0;0.33;0.66;1"
                                values="0.005 0.003;0.01 0.009;0.008 0.004;0.005 0.003" repeatCount="indefinite" />
                        </feTurbulence>
                        <feDisplacementMap in="SourceGraphic">
                            <animate attributeName="scale" dur="20s" keyTimes="0;0.25;0.5;0.75;1"
                                values="45;55;75;55;45" repeatCount="indefinite" />
                        </feDisplacementMap>
                    </filter>
                </defs>
            </svg>
        </div>
        <div id="blinds">
            <div class="shutters">
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
                <div class="shutter"></div>
            </div>
            <div class="vertical">
                <div class="bar"></div>
                <div class="bar"></div>
            </div>
        </div>
    </div>
    <div id="progressive-blur">
        <div></div>
        <div></div>
        <div></div>
        <div></div>
    </div>
</div>
    <article id="writing">
        <p id="nav">
            <a href='/'>Home</a> ~
            <a href='/writing'>Writing</a> ~
            <a href='/reading'>Reading</a>
        </p>
        <hr>
        <div class='content'>
            <div class='front-matter'>
                <div class='wrap'>
                    <h1>Virology [UFO]</h1>
                    <p><i>Notes from Virology Lectures by Vincent Racaniello</i></p>
                    <div class='bylines'>
                        <div class='byline'>
                            <h3>Published</h3>
                            <p>28 May 2020</p>
                        </div>
                    </div>
                    <div class='clear'></div>
                </div>
            </div>
            <hr>
            <div class='wrap-article'>
                <p><em>I got into learning some Virology during the COVID-19 pandemic and lost interest after a few lectures. Nevertheless, here is everything I learnt until now. This file was originally hosted on GitHub<sup id="fnref:1"><a href="#fn:1" class="footnote" rel="footnote" role="doc-noteref">1</a></sup></em></p>

<h2 id="lecture-1-what-is-a-virus">Lecture 1: What is a Virus</h2>

<ul>
  <li>Just like we have a microbiome, we also have a virome.</li>
  <li>The human genome has 3.2 billion bases out of which 8.3% are LTR transposons (remainders of retroviruses).</li>
  <li>
    <p>Amazingly, the vast majority of the viruses that infect us have little to no impact on our health/well-being. Why not?</p>

    <ul>
      <li>
        <p>Most viruses just pass through us. (Many non-animal viruses are injested with food)</p>
      </li>
      <li>Beneficial viruses.</li>
      <li>We have an amazing immune system.</li>
    </ul>
  </li>
  <li><a href="https://www.ncbi.nlm.nih.gov/pubmed/15831942"><strong>Use of polyomavirus to track <em>Homo sapiens</em> migrations patterns in prehistoric times</strong></a></li>
</ul>

<h3 id="course-goals">Course Goals</h3>

<ol>
  <li>See the ‘big picture’ of virology.</li>
  <li>Virology is an integrated discipline, not an isolated collection of viruses, diseases and genes.</li>
  <li>Fundamentals of the viruses.</li>
</ol>

<h3 id="what-is-a-virus">What is a Virus</h3>

<blockquote>
  <p>A virus is an infectious, obligate, intracellular parasite comprising genetic material (DNA/RNA), often surrounded by a protein coat, sometimes a membrane.</p>
</blockquote>

<table>
  <thead>
    <tr>
      <th>Term</th>
      <th>Meaning</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Infectious</td>
      <td>Viruses can move from host to host.</td>
    </tr>
    <tr>
      <td>Obligate Intracellular</td>
      <td>Viruses have to get inside a cell in order to reproduce.</td>
    </tr>
    <tr>
      <td>Parasite</td>
      <td>Occupies a cell and takes essential stuff (nutrients), thereby damaging it.</td>
    </tr>
    <tr>
      <td>Protein shell/membrane</td>
      <td><strong>Often</strong> surrounded by a protein shell. Viroids don’t have protein coats and some viruses (coronaviruses) have membranes</td>
    </tr>
  </tbody>
</table>

<ul>
  <li>
    <p>We need to avoid anthropomorphic analyses of viruses. They don’t think, they don’t have goals etc. <strong>The only selective force for a virus is to find a new host</strong>.</p>
  </li>
  <li>
    <p>Viruses are not all small. Some of the more recently discovered ones are relatively big.</p>
  </li>
  <li>
    <p>Viruses replicate by assembly of pre-formed components into many particles as opposed to binary fission like normal cells or bacteria. This gives rise to an incubation period where nothing seems to happen.</p>
  </li>
</ul>

<div class="figure">
    <img src="https://s3-us-west-2.amazonaws.com/courses-images/wp-content/uploads/sites/1094/2016/11/03154547/OSC_Microbio_06_02_growth.jpg" />
    <div class="caption">
        <span class="caption-label">Figure 1.</span>Eclipse Period for Viruses
    </div>
</div>

<ul>
  <li><strong>How old are viruses?</strong></li>
</ul>

<h3 id="virus-classification">Virus Classification</h3>

<ul>
  <li>
    <p>Nature and symmetry of nucleic acid in virion.</p>
  </li>
  <li>
    <p>Symmetry of protein shell (capsid).</p>
  </li>
  <li>
    <p>Presence/absence of lipid membrane (envelope).</p>
  </li>
  <li>
    <p>Dimensions of virus particle.</p>
  </li>
</ul>

<p>There is an underlying simplicity and order to viruses because of two simple facts:</p>

<ol>
  <li>
    <p>All viral genomes are obligate molecular parasites that can only function after they replicate in a cell.</p>
  </li>
  <li>
    <p>All viruses must make <code class="language-plaintext highlighter-rouge">mRNA</code> that can be translated by host ribosomes (they are all parasites of the host protein synthesis machinery).</p>
  </li>
</ol>

<h3 id="additional-reading">Additional Reading</h3>

<ol>
  <li>Principles of Virology, Flint et al., Volume 1, Chapter 1</li>
  <li><a href="https://www.virology.ws/2019/08/22/the-evolving-concept-of-virus/">The Evolving Concept of the Virus</a></li>
  <li><a href="https://learn.genetics.utah.edu/content/cells/scale/">Cell Size and Scale</a></li>
  <li><a href="https://www.virology.ws/2013/08/01/pandoravirus-bigger-and-unlike-anything-seen-before/">Pandoravirus</a></li>
</ol>

<h2 id="lecture-2-the-infectious-cycle">Lecture 2: The Infectious Cycle</h2>

<p>The Infectious Cycle is everything that happens from when a virus attaches to a cell and new viruses come out at the end of the cycle. Other names: reproductive cycle, replicative cycle.</p>

<p>The steps of the infectious cycle are:</p>

<ol>
  <li><strong>Attachment and entry</strong>: Viruses attach to very specific receptors on the cell surface and are brought into the cell.</li>
  <li><strong>Translation</strong>: The <code class="language-plaintext highlighter-rouge">mRNA</code> is made and translated into proteins.</li>
  <li><strong>Genome replication</strong>: Multiplication of viral genomes.</li>
  <li><strong>Assembly</strong>: Viral proteins + viral genomes</li>
  <li><strong>Release</strong>: Out in search of a new cell</li>
</ol>

<p>Some important definitions:</p>

<ul>
  <li><strong>Susceptible cell</strong>: Cell has a functional receptor for a given virus - <em>the cell may or may not be able to support viral replication</em>.</li>
  <li><strong>Resistant cell</strong>: Cell has no receptor - <em>it may or may not be able to support viral replication</em>.</li>
  <li><strong>Permissive cell</strong>: Cell has the capacity to replicate virus - <em>it may or may not be susceptible</em>.</li>
  <li><strong>Susceptible and Permissive cell</strong>: The only kinds of cells that can take a virus particle and replicate it.</li>
</ul>

<div class="figure">
    <img src="https://s3-us-west-2.amazonaws.com/courses-images/wp-content/uploads/sites/1094/2016/11/03154532/OSC_Microbio_06_02_RepAnVirus.jpg" />
    <div class="caption">
        <span class="caption-label">Figure 2.</span>Virus Infectious Cycle
    </div>
</div>

<h3 id="how-does-one-know-if-a-particular-cell-is-infected">How does one know if a particular cell is infected</h3>

<p>One way is to look at them through a light microscope and looking for the <a href="https://en.wikipedia.org/wiki/Cytopathic_effect"><em>cytopathic effect</em></a> (CPE).</p>

<div class="figure">
    <img src="https://www.researchgate.net/profile/Amr_Gamil2/publication/275258175/figure/fig1/AS:294613517651971@1447252744541/Cytopathic-effects-in-CHSE-cells-Development-of-cytopathic-effect-in-CHSE-214-cells.png" />
    <div class="caption">
        <span class="caption-label">Figure 3.</span>HeLa Cells infected with Poliovirus
    </div>
</div>

<p>Another way is to look for the formation of the <a href="https://en.wikipedia.org/wiki/Syncytium"><em>syncytia</em></a>. These are just two ways of identification and there are many others depending on the type of virus particle involved.</p>

<h3 id="how-many-viruses-are-there-in-the-sample">How many viruses are there in the sample</h3>

<p>We’ve now identified that the virus is replicating and isolated the supernatant. How do we know how much virus is there in the supernatant?</p>

<ul>
  <li>
    <p>There are two ways to do this, we can either measure infectivity or we can measure physical properties of virus particles and their components.</p>
  </li>
  <li>
    <p>We can use a <a href="https://en.wikipedia.org/wiki/Virus_quantification#Plaque_assay"><em>plaque assay</em></a> and quantify the virus. <em>Note: The Wikipedia link has all that we need to know about the method and reagants used.</em> The agar is important because it restricts the movement of the virus, if we just had fluid, the virus would move too fast for us to view the plaque formation (restricts viral diffusion after lysis of infected cells).</p>
  </li>
</ul>

<p>Sometimes, the viruses don’t kill the cells and we can’t see the plaques. So we add a gene to their genome that adds a colour and count the coloured plaques instead.</p>

<h3 id="how-many-viruses-are-needed-to-form-a-plaque">How many viruses are needed to form a plaque</h3>

<p>If we are going to measure the virus titer by a plaque assay, we have to make sure that one virus will make a plaque. We use a dose-response curve to solve this problem.</p>
<div class="figure">
    <img src="https://www.virology.ws/wp-content/uploads/2009/07/dose-response-plaque-assay-1018x1024.jpg" />
    <div class="caption">
        <span class="caption-label">Figure 4.</span>Dose-Response Curve
    </div>
</div>

<h3 id="what-do-we-do-for-viruses-that-do-not-form-a-plaque">What do we do for viruses that do not form a plaque</h3>

<ul>
  <li>
    <p>There is one method called the Endpoint dilution assay. Serial dilutions of a virus stock are prepared and inoculated onto replicate cell cultures, the number of cell cultures that are infected is then determined for each virus dilution, usually by looking for cytopathic effect.</p>
  </li>
  <li>
    <p>After an incubation period, plates that display cytopathic effects are scored with a <code class="language-plaintext highlighter-rouge">+</code>. At high dilutions, none of the cell cultures are infected because no particles are present. At low dilutions, every cell culture is infected. We need to find the dilution of virus at which 50% of the cell cultures are infected.</p>
  </li>
</ul>

<p>The results are expressed as 50% lethal dose (LD<sub>50</sub>) per ml or 50% paralytic dose (PD<sub>50</sub>) per ml when lethality or paralysis are used as end points.</p>

<h3 id="particlepfu-ratio">Particle/PFU Ratio</h3>

<ul>
  <li>Particle/PFU Ratio = # of physical particles / # of infectious particles</li>
</ul>

<p>The term particle/PFU ratio refers to the number of viral particles required to form one plaque in a plaque assay. It is a measure of the efficiency by which a virus infects cultured cells. <strong>Not all virus particles are infectious</strong>. For example, the poliovirus has a ratio between 30 to 1000 while Semliki forest virus has a ratio of 1 or 2.</p>

<ul>
  <li>
    <p>A single particle <em>can</em> initiate infection.</p>
  </li>
  <li>
    <p>Not all viruses are successful though</p>
    <ul>
      <li>Damaged particles (broken during assembly et al.)</li>
      <li>Mutations</li>
    </ul>
  </li>
</ul>

<h4 id="incubation-period-and-eclipse-period-are-not-the-same">INCUBATION PERIOD AND ECLIPSE PERIOD ARE NOT THE SAME</h4>

<h3 id="single-step-and-multi-step-growth-curves">Single Step and Multi Step Growth Curves</h3>

<p>If we infect all the cells in one go, the virus will have a traditional growth curve with an eclipse period and stuff. But what happens in real life is that the virus particles never attack all the cells at one time. So we have, in reality, a multi-step growth curve that has multiple bursts with multiple eclipse periods.</p>

<ul>
  <li>Synchronous infection is the key to one-step growth cycles. But how do we know if we’ve infected all the cells?</li>
</ul>

<h4 id="multiplicity-of-infection-moi">Multiplicity of Infection (MOI)</h4>

<ul>
  <li>Number of infectious particles added per cell.</li>
  <li>Amount of virus (PFU) / # of cells</li>
</ul>

<p>This is different from the number of particles each cell recieves. If 10<sup>7</sup> virus particles are added to 10<sup>6</sup> cells. We have an MOI of 10, but each cell doesn’t get 10 virus particles.</p>

<ul>
  <li>Infection depends on the random collision of virus particles and cells. Best described by the Poisson Distribution.</li>
</ul>

<h3 id="additional-reading-1">Additional Reading</h3>

<ol>
  <li>Principles of Virology, Flint et al., Volume 1, Chapter 2</li>
  <li><a href="https://www.virology.ws/2009/12/10/influenza-virus-growth-in-eggs/">Influenza virus growth in eggs</a></li>
  <li><a href="https://www.virology.ws/2009/02/09/the-amazing-hela-cells-of-henrietta-lacks/">The amazing HeLa cells of Henrietta Lacks</a></li>
  <li><a href="https://www.youtube.com/watch?v=tGVDYGxqtJ0">Counting Viruses</a></li>
  <li><a href="https://www.virology.ws/2017/02/17/viral-rna-is-not-infectious-virus/">Viral RNA is not infectious virus!</a></li>
</ol>

<h2 id="lecture-3-genomes-and-genetics">Lecture 3: Genomes and Genetics</h2>

<p><a href="https://www.biorxiv.org/content/10.1101/2020.01.22.914952v1">Discovery of a novel coronavirus associated with the recent pneumonia outbreak in humans and its potential bat origin</a>. <em>The following line is from the paper, where we can see real-time applications of what we learned in the previous lecture</em></p>

<blockquote>
  <p>We then successfully isolated the virus (named nCoV-2019 124 BetaCoV/Wuhan/WIV04/2019), in Vero and Huh7 cells using BALF sample from ICU-06 patient. Clear cytopathogenic effects were observed in cells after three days 126 incubation.</p>
</blockquote>

<h3 id="the-hershey-chase-experiment">The Hershey-Chase Experiment</h3>

<p>Initially, the scientists studying virus particles thought that proteins carried genetic information. This was based on the belief that proteins were more complex than <code class="language-plaintext highlighter-rouge">DNA</code>. <em>For example, <code class="language-plaintext highlighter-rouge">DNA</code> in humans have only 4 bases, how could it carry that much of information</em>.</p>

<ul>
  <li>
    <p>Viruses were known to be composed of a protein shell and <code class="language-plaintext highlighter-rouge">DNA</code>, so they chose to uniquely label each with a different elemental isotope.</p>
  </li>
  <li>
    <p>Since phosphorus is contained in <code class="language-plaintext highlighter-rouge">DNA</code> but not amino acids, radioactive phosphorus-32 was used to label the <code class="language-plaintext highlighter-rouge">DNA</code> contained in the T2 phage. Radioactive sulfur-35 was used to label the protein sections of the T2 phage, because sulfur is contained in protein but not <code class="language-plaintext highlighter-rouge">DNA</code>.</p>
  </li>
</ul>

<div class="figure">
    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/56/Tevenphage.svg/330px-Tevenphage.svg.png" />
    <div class="caption">
        <span class="caption-label">Figure 5.</span>Structural Overview of T2 Phage
    </div>
</div>

<ul>
  <li>
    <p>Hershey and Chase inserted the radioactive elements in the bacteriophages by adding the isotopes to separate media within which bacteria were allowed to grow for 4 hours before bacteriophage introduction.</p>
  </li>
  <li>
    <p>When the bacteriophages infected the bacteria, the progeny contained the radioactive isotopes in their structures.</p>
  </li>
  <li>
    <p>This procedure was performed once for the sulfur-labeled phages and once for phosphorus-labeled phages.</p>
  </li>
  <li>
    <p>The labeled progeny were then allowed to infect unlabeled bacteria. The phage coats remained on the outside of the bacteria, while genetic material entered.</p>
  </li>
  <li>
    <p>Disruption of phage from the bacteria by agitation in a blender followed by centrifugation allowed for the separation of the phage coats from the bacteria. These bacteria were lysed to release phage progeny.</p>
  </li>
</ul>

<p><em>Note: We now know that some phages completely enter the bacteria like <code class="language-plaintext highlighter-rouge">E. Coli</code> but it was a fortunate coincidence that the T2 phages didn’t do so.</em></p>

<ul>
  <li><strong>The progeny of the phages that were labeled with radioactive phosphorus remained labeled, whereas the progeny of the phages labeled with radioactive sulfur were unlabeled</strong>. Thus, the Hershey–Chase experiment helped to confirm that <code class="language-plaintext highlighter-rouge">DNA</code>, not protein, is the genetic material.</li>
</ul>

<div class="figure">
    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/3/32/Hershey_Chase_experiment.png/536px-Hershey_Chase_experiment.png" />
    <div class="caption">
        <span class="caption-label">Figure 6.</span>Hershey-Chase Experiment
    </div>
</div>

<p>Similar Experiment: The Fraenkel-Conrat Experiment on Tobacco Mosaic Virus.</p>

<h3 id="number-of-viral-genomes">Number of Viral Genomes</h3>

<p>There are thousands of different virions, and it seems that there is an infinite complexity of infections. But the number of viral genomes is finite, in fact, it is my favourite number: <strong>7</strong>.</p>

<p>One of the reasons for this is all viruses or viral genomes have to make <code class="language-plaintext highlighter-rouge">mRNA</code> that can be translated by the host ribosomes. This constraint causes the limit. <em>I have to admit, this doesn’t exactly give a proper answer. What is the relation between the constraint on the <code class="language-plaintext highlighter-rouge">mRNA</code> and the number of viral genomes? I couldn’t find an answer anywhere and I’ve written a mail to Prof. Racaniello. I’ll update it once I get an answer.</em></p>

<h4 id="the-baltimore-system-simplifying-virus-classification"><a href="https://www.virology.ws/2009/08/12/simplifying-virus-classification-the-baltimore-system/">The Baltimore System: Simplifying Virus Classification</a></h4>

<p>Virologist <a href="https://en.wikipedia.org/wiki/David_Baltimore">David Baltimore</a> devised an alternative classification scheme that takes into account the nature of the viral nucleic acid.</p>

<p>All viruses <strong>must</strong> direct the synthesis of <code class="language-plaintext highlighter-rouge">mRNA</code> to produce proteins. No viral genome encodes a complete system for translating proteins; therefore all viral protein synthesis is completely dependent upon the translational machinery of the cell.</p>

<div class="figure">
    <img src="https://www.virology.ws/wp-content/uploads/2009/08/baltimore-classification.jpg" />
    <div class="caption">
        <span class="caption-label">Figure 7.</span>Baltimore System
    </div>
</div>

<table>
  <thead>
    <tr>
      <th>Class</th>
      <th>Type</th>
      <th>Remarks</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>Double-stranded <code class="language-plaintext highlighter-rouge">DNA</code></td>
      <td>We can make <code class="language-plaintext highlighter-rouge">mRNA</code> from this</td>
    </tr>
    <tr>
      <td>2</td>
      <td>Single-stranded <code class="language-plaintext highlighter-rouge">DNA</code></td>
      <td>Double-stranded <code class="language-plaintext highlighter-rouge">DNA</code> is made from this and then to <code class="language-plaintext highlighter-rouge">mRNA</code>. <strong>Single-stranded <code class="language-plaintext highlighter-rouge">DNA</code> is not a template for <code class="language-plaintext highlighter-rouge">mRNA</code> synthesis</strong></td>
    </tr>
    <tr>
      <td>3</td>
      <td>Double-stranded <code class="language-plaintext highlighter-rouge">RNA</code></td>
      <td>Both the messenger and anti-messenger <code class="language-plaintext highlighter-rouge">RNA</code></td>
    </tr>
    <tr>
      <td>4</td>
      <td><code class="language-plaintext highlighter-rouge">+ RNA</code></td>
      <td>In some cases, we can translate it directly but we need to go through a <code class="language-plaintext highlighter-rouge">- RNA</code> to make more of it.</td>
    </tr>
    <tr>
      <td>5</td>
      <td><code class="language-plaintext highlighter-rouge">- RNA</code></td>
      <td>It must first be transcribed into a positive-sense <code class="language-plaintext highlighter-rouge">RNA</code> that acts as an <code class="language-plaintext highlighter-rouge">mRNA</code>.</td>
    </tr>
    <tr>
      <td>6</td>
      <td><code class="language-plaintext highlighter-rouge">+ RNA</code></td>
      <td>Unusual in the sense that it is converted to <code class="language-plaintext highlighter-rouge">DNA</code> which is then converted to double-stranded <code class="language-plaintext highlighter-rouge">DNA</code></td>
    </tr>
  </tbody>
</table>

<p><em>Note: When originally conceived, the Baltimore scheme encompassed six classes of viral genome, as shown in the figure. Subsequently the gapped <code class="language-plaintext highlighter-rouge">DNA</code> genome of hepadnaviruses (e.g. hepatitis B virus) was discovered.</em></p>

<p>A few definitions that will help make more sense follow:</p>

<ul>
  <li><code class="language-plaintext highlighter-rouge">mRNA</code> is always the <code class="language-plaintext highlighter-rouge">+</code> strand.</li>
  <li><code class="language-plaintext highlighter-rouge">DNA</code> of equivalent polarity is also the <code class="language-plaintext highlighter-rouge">+</code> strand.</li>
  <li><code class="language-plaintext highlighter-rouge">RNA</code> and <code class="language-plaintext highlighter-rouge">DNA</code> complements of <code class="language-plaintext highlighter-rouge">+</code> strands are <code class="language-plaintext highlighter-rouge">-</code> strands.</li>
  <li>Not all <code class="language-plaintext highlighter-rouge">+ RNA</code> is <code class="language-plaintext highlighter-rouge">mRNA</code>!</li>
</ul>

<h4 id="in-conclusion-the-seven-classes-of-viral-genomes">In conclusion: The Seven Classes of Viral Genomes</h4>

<ul>
  <li>
    <p>Double Stranded <code class="language-plaintext highlighter-rouge">DNA</code></p>
  </li>
  <li>
    <p>Gapped Double Stranded <code class="language-plaintext highlighter-rouge">DNA</code></p>
  </li>
  <li>
    <p>Single Stranded <code class="language-plaintext highlighter-rouge">DNA</code></p>
  </li>
  <li>
    <p>Double Stranded <code class="language-plaintext highlighter-rouge">RNA</code></p>
  </li>
  <li>
    <p>Single Stranded <code class="language-plaintext highlighter-rouge">+ RNA</code></p>
  </li>
  <li>
    <p>Single Stranded <code class="language-plaintext highlighter-rouge">- RNA</code></p>
  </li>
  <li>
    <p>Single Stranded <code class="language-plaintext highlighter-rouge">+ RNA</code> with <code class="language-plaintext highlighter-rouge">DNA</code> intermediate</p>
  </li>
</ul>

<div class="figure">
    <img src="https://microbeonline.com/wp-content/uploads/2016/06/Baltimore-System-of-Classification-of-Viruses.bmp" />
    <div class="caption">
        <span class="caption-label">Figure 8.</span>Baltimore System
    </div>
</div>

<h3 id="what-information-is-encoded-in-a-viral-genome">What information is encoded in a viral genome</h3>

<p>Gene products and regulatory signals for:</p>

<ul>
  <li>Protein synthesis (mainly giant viruses)</li>
  <li>Replication of the viral genome</li>
  <li>Assembly and packaging of the genome</li>
  <li>Regulation and timing of the replication cycle</li>
  <li>Modulation of host defenses</li>
  <li>Spread to other cells and hosts.</li>
</ul>

<h3 id="what-information-is-not-encoded-in-a-viral-genome">What information is NOT encoded in a viral genome</h3>

<ul>
  <li>No genes encoding the complete protein synthesis machinery</li>
  <li>No genes encoding proteins involved in membrane biosynthesis</li>
  <li>No classical centromeres or telomeres found in standard host chromosomes</li>
  <li>Probably we haven’t found them yet - 90% of giant virus genes are novel.</li>
</ul>

<h3 id="extra-reading">Extra Reading</h3>

<ol>
  <li>Principles of Virology, Flint et al., Volume 1, Chapter 3</li>
  <li><a href="https://www.virology.ws/2009/08/12/simplifying-virus-classification-the-baltimore-system/">Baltimore Classification</a></li>
  <li><a href="https://viralzone.expasy.org/">Viral Zone</a></li>
</ol>

<h2 id="lecture-4-structure-of-viruses">Lecture 4: Structure of Viruses</h2>

<h3 id="functions-of-structural-proteins">Functions of structural proteins</h3>

<h4 id="protection-of-the-genome">Protection of the genome</h4>

<ul>
  <li>Assembly of a stable protective protein shell.</li>
  <li>Specific recognition and packaging of the nucleic acid genome.</li>
  <li>Interaction with host cell membranes to form the envelope.</li>
</ul>

<h4 id="delivery-of-the-genome">Delivery of the genome</h4>

<ul>
  <li>Bind host cell receptors.</li>
  <li>Uncoating of the genome.</li>
  <li>Fusion with cell membranes.</li>
  <li>Transport of genome to the appropriate site.</li>
</ul>

<h3 id="some-definitions">Some definitions</h3>

<ul>
  <li><strong>Subunit</strong>: A single folded polypeptide chain. <em>Denoted by VP1, VP2 etc.</em></li>
  <li><strong>Structural unit</strong> (protomer, asymmetric unit) - Unit from which capsids or nucleocapsids are built; one or more subunits.</li>
  <li><strong>Capsid</strong> (capsa = Latin, box) - Protein shell surrounding genome. <em>It is made up of subunits.</em></li>
  <li><strong>Envelope</strong> (viral membrane) - Host cell-derived lipid bilayer. <em>For some viruses only, this always comes from the host cell because viruses cannot make lipids</em>.</li>
  <li><strong>Nucleocapsid</strong> (core) - Nucleic acid - protein assembly within particle. <em>Nucleocapsid is used against capsid when the it’s a discrete substructure</em>.</li>
</ul>

<h3 id="metastability">Metastability</h3>

<p>An important concept to understand is that virus particles are meta-stable. The capsid proteins have to be very stable in transit as the virus tries to find a new host <em>(Example: Respiratory virus particles have to be very stable in aerosols for effective transfer)</em> but when it finds the right host, the capsid has to give up the genome by becoming unstable.</p>

<p>The metastability of the virions (virus particles) arises from the fact that they haven’t attained the minimum free energy conformation. They first need to surmount the energy barrier and the potential energy is used for disassembly if the host cell gives a proper signal.</p>

<div class="figure">
    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a0/Meta-stability.svg/375px-Meta-stability.svg.png" />
    <div class="caption">
        <span class="caption-label">Figure 9.</span>Metastability
    </div>
</div>

<h4 id="how-is-metastability-achieved">How is metastability achieved</h4>

<ul>
  <li>
    <p>Stability in structure is achieved by creating a <strong>symmetrical</strong> arrangement of many identical proteins to
provide maximal contact.</p>
  </li>
  <li>
    <p>Unstability happens primarily because these proteins aren’t covalently joined and can be taken apart/loosened upon infection.</p>
  </li>
</ul>

<h3 id="symmetry">Symmetry</h3>

<p>There are primarily three kinds of virus particles: particles with helical symmetry (<em>Tobacco Mosaic Virus</em>), icosahedral symmetry (<em>Poliovirus</em>), complex (<em>don’t fit into the above two</em>).</p>

<p>Let’s go back to Watson and Crick who are most famous for discovering the structure of <code class="language-plaintext highlighter-rouge">DNA</code>. <em>For those of you who are into this stuff, I recommend reading The Double Helix, it’s just fantastic</em>.</p>

<p>Watson and Crick observed that most virus particles were spherical or rod-shaped. They also knew that viral genomes were small which meant that particles would be built with many copies of a few proteins.</p>

<ul>
  <li>
    <p>Rod-shaped viruses are identical protein subunits that are distributed with <em>helical symmetry</em>.</p>
  </li>
  <li>
    <p>Spherical viruses are identical protein subunits that are distributed with <em>icosahedral symmetry</em>.</p>
  </li>
</ul>

<h4 id="rules-of-symmetry">Rules of Symmetry</h4>

<ul>
  <li>
    <p><strong>Rule 1</strong>: Each subunit has ‘identical’ bonding contacts with its neighbors. (<em>Repeated interaction of chemically complementary surfaces at the subunit interfaces naturally leads to a symmetric arrangement</em>)</p>
  </li>
  <li>
    <p><strong>Rule 2</strong>: These bonding contacts are usually non-covalent. (<em>Reversible; error-free assembly. This means that if there’s an error in the assembly of the particle, we can reverse it and re-assemble which wouldn’t be possible in case of a covalent bond.</em>)</p>
  </li>
</ul>

<h4 id="symmetry-and-self-assembly">Symmetry and Self-Assembly</h4>

<p>If we have capsid proteins lying around in a cell, they will self-assemble into virus-like particles (VLP’s) and these VLP’s are used in some cases as vaccines. Some examples are the Hepatitis B vaccines, human papilloma virus vaccines which are VLP’s made in yeast.</p>

<h3 id="helical-symmetry">Helical Symmetry</h3>

<ul>
  <li>
    <p>The coat protein subunits form a helical subunit with number of proteins/turn = 3. There are both protein-protein contacts as well as protein-<code class="language-plaintext highlighter-rouge">RNA</code> contacts. (<code class="language-plaintext highlighter-rouge">+ RNA</code> in case of TMV)</p>
  </li>
  <li>
    <p>Animal viruses also have helical symmetry but are always enveloped.</p>
  </li>
</ul>

<div class="figure">
    <img src="https://biologynotesonline.com/wp-content/uploads/2024/11/Structure-of-Tobacco-Mosaic-Virus-TMV.jpg" />
    <div class="caption">
        <span class="caption-label">Figure 10.</span>Helical Symmetry in TMV
    </div>
</div>

<h3 id="icosahedral-symmetry">Icosahedral Symmetry</h3>

<p>The important question here is: How can you make a round capsid from proteins with irregular shapes? Watson and Crick answered that by using two important ideas:</p>

<ul>
  <li>
    <p>All round capsids have precise numbers of proteins; multiples of 60 are common (60, 180, 240, 960)</p>
  </li>
  <li>
    <p>Spherical viruses come in many sizes, but capsid proteins are 20-60 kDa average.</p>
  </li>
</ul>

<p>An icosahedron is a solid with 20 faces, each an equilateral triangle. It has five, three and two-fold axes of symmetry (12 each). It also allows formation of a closed shell with smallest number (60) of identical subunits.</p>

<h4 id="simple-icosahedral-capsids">Simple icosahedral capsids</h4>

<div class="figure">
    <img src="https://cdn.britannica.com/45/4945-004-C544DBFA/virus-icosahedron-symmetry-axes-Edges-lines-surfaces.jpg" />
    <div class="caption">
        <span class="caption-label">Figure 11.</span>Icosahedral symmetry
    </div>
</div>

<ul>
  <li>Made of 60 identical protein subunits</li>
  <li>The protein subunit is the structural unit</li>
  <li>Interactions of all molecules with their neighbors are identical (head-to-head, tail-to-tail)</li>
  <li>The particles are spherical, not icosahedra! (<em>This means that the particles are just arranged in an icosahedral symmetry but they aren’t flat and triangular, so they look more spherical</em>)</li>
</ul>

<h4 id="t-triangulation-number"><a href="http://www.virology.wisc.edu/virusworld/tri_number.php">T (Triangulation) Number</a></h4>

<p>It is the number of facets in each of the faces of this particle.</p>

<h3 id="enveloped-virions">Enveloped Virions</h3>

<p>We know that capsids can be covered by host membranes. The envelope is a lipid bilayer derived from host cell as the viral genome does not encode lipid synthetic machinery.</p>

<p>Envelopes are usually acquired by budding of nucleocapsid through a cellular membrane. Can be any cell membrane, but is virus-specific.</p>

<p>Nucleocapsids inside the envelope may have helical or icosahedral symmetry.</p>

<h3 id="more-reading">More Reading</h3>

<ol>
  <li>Principles of Virology, Flint et al., Volume 1, Chapter 4</li>
  <li><a href="https://youtu.be/qLAEUvlVmqY">Buckyball viruses</a></li>
  <li><a href="https://www.virology.ws/2009/03/06/virus-images-at-viperdb/">Virus images at VIPERdb</a></li>
</ol>

<h2 id="lecture-5-attachment-and-entry">Lecture 5: Attachment and Entry</h2>

<ul>
  <li>
    <p>Viruses are obligate intracellular parasites but they’re too big to just diffuse across the plasma membrane.</p>
  </li>
  <li>
    <p>They need to find the <em>right</em> cell receptor. They first adhere to the cell surface via random collisions and electrostatics, then they attach to specific receptor molecules on cell surface and transfer the genome inside the cell.</p>
  </li>
</ul>

<p>There are some criteria for identifying cell receptors for viruses:</p>

<ul>
  <li>
    <p>We have to show that the receptor actually binds to the virus particle.</p>
  </li>
  <li>
    <p>There exists an antibody to the receptor that blocks infection.</p>
  </li>
  <li>
    <p>There is a receptor gene that confers susceptibility and its disruption blocks infection.</p>
  </li>
</ul>

<p>Sometimes, different viruses may bind to the same receptor. And other times, viruses of the same family bind to different receptors.</p>

<h3 id="how-do-viruses-attach-to-receptors">How do viruses attach to receptors</h3>

<p>For foreign particles, entry into viruses happens in many ways like <a href="https://en.wikipedia.org/wiki/Phagocytosis">phagocytosis</a>, <a href="https://en.wikipedia.org/wiki/Endocytosis">endocytosis</a> and <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2016.00300/full#h6">surfing</a>. We are mostly interested in endocytosis.</p>

<p>Important point: <strong>To reiterate, the cytoplasm is extremely crowded, and movement of large protein complexes will not take place via diffusion</strong></p>

<h4 id="class-1-fusion-proteins">Class 1 Fusion Proteins</h4>

<ul>
  <li>
    <p>Perpendicular to membrane - spikes.</p>
  </li>
  <li>
    <p>Mostly α-helical.</p>
  </li>
  <li>
    <p>Form trimers.</p>
  </li>
</ul>

<h4 id="class-2-fusion-proteins">Class 2 Fusion Proteins</h4>

<ul>
  <li>
    <p>Mostly β-sheet.</p>
  </li>
  <li>
    <p>Form dimers.</p>
  </li>
  <li>
    <p>Parallel to the membrane.</p>
  </li>
</ul>

<h4 id="regulation-of-fusion">Regulation of fusion</h4>

<ul>
  <li>
    <p>Fusion must not occur in the wrong location.</p>
  </li>
  <li>
    <p>Neutral pH (plasma membrane)</p>

    <ul>
      <li>Second protein receptor interaction.</li>
    </ul>
  </li>
  <li>
    <p>Low pH fusion.</p>

    <ul>
      <li>
        <p>Proteolytic cleavage activates the fusion protein for cleavage.</p>
      </li>
      <li>
        <p>Endosome fusion receptor.</p>
      </li>
    </ul>
  </li>
</ul>

<h3 id="some-more-reading">Some more Reading</h3>

<ol>
  <li>Principles of Virology, Flint et al., Volume 1, Chapter 5</li>
  <li><a href="https://www.virology.ws/2019/02/14/a-portal-for-rna-exit/">A portal for RNA exit</a></li>
  <li><a href="https://www.virology.ws/2018/02/01/a-human-rhinovirus-in-chimpanzees/">A human rhinovirus in chimpanzees</a></li>
</ol>

<h2 id="lecture-6-rna-directed-rna-synthesis">Lecture 6: <code class="language-plaintext highlighter-rouge">RNA</code> directed <code class="language-plaintext highlighter-rouge">RNA</code> synthesis</h2>

<p><em>I lost interest after this point, I hope to return sometime soon…</em></p>

<hr />
<div class="footnotes" role="doc-endnotes">
  <ol>
    <li id="fn:1">
      <p><a href="https://github.com/kyscg/Paper2Pulp/blob/master/notes/Virology%20with%20Vincent%20Racaniello.md">Virology with Vincent Racaniello</a> <a href="#fnref:1" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
  </ol>
</div>


                <script src="https://utteranc.es/client.js" repo="kyscg/kyscg.github.io" issue-term="url"
                    label="comments" theme="github-light" crossorigin="anonymous" async></script>
            </div>
        </div>
    </article>
</body>

</html>
